RecoTracker/FinalTrackSelectors/python/trackSelectionLwtnn_cfi.py

```diff
@@ -0,0 +1,5 @@
+from RecoTracker.FinalTrackSelectors.lwtnnESProducer_cfi import lwtnnESProducer as _lwtnnESProducer
+trackSelectionLwtnn = _lwtnnESProducer.clone(
+    ComponentName = "trackSelectionLwtnn",
+    fileName = "RecoTracker/FinalTrackSelectors/data/LWTNN_network_v3.json",
```

Comment:
is this meant to stay as v3?
what was the purpose of v7

Reply:
You are correct, it should be v7. Fixed it now.